src/Alert/Alert.component.tsx

```diff
@@ -0,0 +1,182 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import classNames from 'classnames';
+import styled, {
+    css,
+    FlattenInterpolation,
+    ThemeProps,
+} from '@xstyled/styled-components';
+import { th } from '@xstyled/system';
+import { transparentize, darken } from 'polished';
+// COMPONENTS
+import { Close } from '../Icon';
+import { Typography } from '../Typography';
+import { Button } from '../Button';
+import { get } from '../utils/get/get';
+
+export enum AlertTypes {
+    success = 'SUCCESS',
+    info = 'INFO',
+    warning = 'WARNING',
+    error = 'ERROR',
+}
+
+type AlertType =
+    | 'success'
+    | 'info'
+    | 'warning'
+    | 'error'
+    | AlertTypes.success
+    | AlertTypes.info
+    | AlertTypes.warning
+    | AlertTypes.error;
+
+interface AlertProps extends React.HTMLAttributes<HTMLDivElement> {
+    type: AlertType;
+    className?: string;
+    message?: string;
+    description?: string;
+    closable?: boolean;
+    icon?: JSX.Element;
+    onClose?: (event?: Event) => void;
+}
+
+const renderAlertTheme = (
+    type: AlertType
+): FlattenInterpolation<ThemeProps<AlertProps>> => {
+    return css`
+        ${({ theme }: any) =>
+            css({
+                backgroundColor: transparentize(
+                    0.75,
+                    get(theme, `colors.${type.toLowerCase()}`)
+                ),
+                border: `solid thin ${transparentize(
+                    0.4,
+                    get(theme, `colors.${type.toLowerCase()}`)
```

Comment:
There are semantic border colors for `error` and `success`. We could add the rest (`info`, `warning`, actually define them (currently set to placeholder of `ash.dark`) and use those.

Reply:
We could do the same for `colors.backgrounds` as well, if you like (e.g. `colors.backgrounds.error`.)

Reply:
There's some complexity that arises because we'll need the same color but different levels of transparency in this component. Where would you put the transparency for the bg/border values? Or, would you reduce the complexity of those to use hex colors that render roughly the same hue?

Reply:
I just pushed a theme where I add the transparent colors to the theme to prevent introducing new hues to the color palette.